data/json/recipes/other/tool.json

```diff
@@ -136,9 +136,10 @@
     "category": "CC_OTHER",
     "subcategory": "CSC_OTHER_TOOLS",
     "skill_used": "fabrication",
-    "difficulty": 3,
-    "time": "20 m",
+    "difficulty": 1,
+    "time": "15 m",
     "autolearn": true,
+    "proficiencies": [ { "proficiency": "prof_knapping", "required": false, "time_multiplier": 4.0 } ],
```

Comment:
```suggestion
    "proficiencies": [ { "proficiency": "prof_knapping", "required": true } ],
```

Reply:
"Experiments in knapping have demonstrated the relative ease with which a hand axe can be made" ... "No long or demanding apprenticeship is necessary to learn the necessary techniques."  If I don't know knapping, I can wing it.  I'll mess up, start over, be slower, produce less quality; but I expect to have a usable, cuttable edge in the end.

Here's an alternative idea: 3 minutes if you have proficiency in knapping.  If no prof_knapping, x20 = 1 hour.